assets/css/components/button.scss

```
@@ -0,0 +1,81 @@
+.btn {
+  @extend %font-ui-text-base-good;
+  padding: 0 10px;
```

```suggestion
  padding: 4px 12px;
```

Sorry, changed my mind after seeing it implemented!